code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1594,6 +1594,7 @@
 		dna.real_name = real_name
 
 	species.handle_post_spawn(src)
+	species.handle_dna(src) //Give them whatever special dna business they got.

```

Comment:
I'll go ahead and review this to see if this line's even necessary, I'd rather not remove it from set_species because that'd re-break powers applying to people getting their species set through C.M.A.